http/http_client.hpp

```diff
@@ -14,31 +14,18 @@
 // limitations under the License.
 */
 #pragma once
-
-#include "async_resolve.hpp"
-#include "http_response.hpp"
-
-#include <boost/asio/connect.hpp>
-#include <boost/asio/io_context.hpp>
 #include <boost/asio/ip/address.hpp>
 #include <boost/asio/ip/basic_endpoint.hpp>
-#include <boost/asio/ip/tcp.hpp>
-#include <boost/asio/ssl/context.hpp>
-#include <boost/asio/ssl/error.hpp>
```

Comment:
These don't look right, I still see SSL error on line 256
            if (ec && ec != boost::asio::ssl::error::stream_truncated)
            
Run include what you use?

Reply:
this file is copied from 1030 and I don't  see any compilation and runtime errors